encord/collection.py

```diff
@@ -171,3 +178,85 @@ def list_items_include_inaccessible(
                 yield StorageItem(api_client=self._client, orm_item=item)
             else:
                 yield StorageItemInaccessible(orm_item=item)
+
+    def add_items(self, storage_item_id_list: List[Union[UUID, str]]) -> CollectionBulkItemResponse:
```

Comment:
very small nitpick, but we don't seem to use `_list` suffix anywhere else in the api, and just use plural, like `storage_item_ids`, should we change it here too?
Not a blocker though, but a thing to consider.

Reply:
Updated